src/olympia/addons/management/commands/process_addons.py

```diff
@@ -42,6 +43,9 @@
             Q(**{current_autoapprovalsummary + 'verdict': amo.AUTO_APPROVED}) &
             ~Q(**{current_autoapprovalsummary + 'confirmed': True})
         ]},
+    'constantly_recalculate_post_review_weight': {
+        'method': recalculate_post_review_weight,
+        'qs': get_recalc_needed_filters()},
```

Comment:
nit: IMHO this function doesn't need to live outside this file (as least not for the moment)

Reply:
Fixed in 6a6852231c